app.js

```diff
@@ -32,15 +32,11 @@ global.toId = function (text) {
 };
 
 for (let server in Config.servers) {
-	if (typeof Config.servers[server].rooms === Array) {
-		for (let room in Config.servers[server].rooms) {
-			Config.servers[server].rooms[room] = toId(Config.servers[server].rooms[room]);
-		}
+	if (Config.servers[server].rooms instanceof Array) {
+		Config.servers[server].rooms = Config.servers[server].rooms.map(toId);
```

Comment:
Essentially, yes, but it takes fewer lines of code to do the same thing (from what I've tested) without using for...in on an Array, I suppose.